.github/workflows/dev-server-ci.yml

```
@@ -0,0 +1,60 @@
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: ./server
+          file: ./server/Dockerfile
+          push: true
+          tags: |
+            ghcr.io/${{ github.repository }}:latest
+            ghcr.io/${{ github.repository }}:${{ github.sha }}
+          labels: |
+            org.opencontainers.image.source=${{ github.repository_url }}
+            org.opencontainers.image.revision=${{ github.sha }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)